src/containers/App/tests/reducer.test.js

```diff
@@ -11,7 +11,11 @@ describe('containers.App.reducer', () => {
       loading: false,
       error: false,
       currentUser: '',
-      userData: fromJS({}),
+      userData: fromJS({
```

Comment:
can you run command `npm run test:unit` ? Does it fail? If no, why is your Jest not working?